.github/workflows/build-and-release-clients.yml

```diff
@@ -0,0 +1,71 @@
+name: Build and release OneLife client
+on:
+  push:
+    tags:
+      - "*"
+permissions:
+  contents: write
+
+jobs:
+  build:
+    name: Publish clients for Linux and Windows (unsigned)
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Cache APT packages
+        uses: awalsh128/cache-apt-pkgs-action@v1.2.4
+        with:
+          packages: rsync wget unzip git imagemagick xclip \
+            libglu1-mesa-dev libgl1-mesa-dev libsdl1.2-dev mingw-w64 \
+            build-essential libpng-dev
+
+      - name: Checkout compile scripts repository
+        uses: actions/checkout@v3
+        with:
+          repository: risvh/miniOneLifeCompile
+          # NOTE: Update this accordingly before release!
+          ref: b92fb84ba587e17c3e5758ceb61164a8a9a24b89
```

Comment:
I'd suggest removing the `ref` parameter completely. It's fine to target the default branch of the miniOneLifeCompile repository.